### DISCUSSION TITLE
How to implement adobe analytics?

### DISCUSSION THREAD

**Discussion by Maniraj-Murugan** [vercel/next.js#10679] [Help]
I have raised an issue here https://github.com/zeit/next.js/issues/10678 and also posted a question in stackoverflow, https://stackoverflow.com/questions/60376041/how-to-implement-adobe-analytics-in-reactjs but I couldn't get any relevant help..

Can anyone help me to implement adobe analytics in nextjs project with appropriate examples?

**Comment by Maniraj-Murugan** [comment]
@timneutkens , Can you provide an example of adobe analytics as you have closed the issue https://github.com/zeit/next.js/issues/10678 raised?

**Reply by timneutkens** [reply]
I've never used Adobe analytics. Last time I inspected a site that uses it the JS bundle for Adobe analytics was about 60% of total JS size though and 2x the size of `react-dom`, so something to watch out for.

**Comment by jamesmosier** [comment]
[Here is an example React app](https://github.com/adobe/aem-sample-we-retail-journal/tree/master/react-app) that should help.

**Comment by Nagarjuna-nxa020-ftr** [comment]
Can anyone help me to implement adobe analytics in nextjs project with appropriate examples?

**Reply by maheshk06** [reply]
@nxa020 you found any solution then please let me know.
